### ISSUE TITLE
Getting the cpu steal time

### ISSUE THREAD

**Issue by Cheyenne55** [sebhildebrandt/systeminformation#842]
**Is your feature request related to a problem? Please describe.**
On Linux OS running on Virtual Machines managed by an Hypervisor there is a metric called *steal time* (good explaination [here](https://www.site24x7.com/learn/linux/cpu-steal-time.html)).

It can be seen when running the *top* command :
![image](https://github.com/sebhildebrandt/systeminformation/assets/14312865/3d983699-cd61-4b6a-ad39-95f0122ffea9)

**Describe the solution you'd like**
I would like to get this value with this lib.

**Describe alternatives you've considered**
Alternative is to get the value from /proc/stat with this command for instance : 
```sh
 awk -v oldidle=$(awk '/cpu / {print $9}' /proc/stat; sleep 1) '/cpu / {perc=100-($9-oldidle)/100 ; printf "%s", perc}' /proc/stat
```

**Additional context**
This is an important metric in Cloud environments because when it is high it means that the VM is under-performing, so we could decide to replace it by a new one (hoping that the next one won't be affected).

**Comment by sebhildebrandt** [comment]
@Cheyenne55 good idea! Will add it in the upcoming days. 

What about "guest" time (Time spent for running a virtual CPU or guest OS under the control of the kernel.)? 

I guess this would then also be interesting from the host perspective ...

**Comment by PublicWorld** [comment]
How about macOS? 😆

**Comment by sebhildebrandt** [comment]
@Cheyenne55 ... I implemented a first version that should return the steal time and guest time. Can you pull the master branch and test it?

```
const si = require('../systeminformation/lib/index.js');
setInterval(() => {
	si.currentLoad().then(res => {
  		console.log(res)
	});
}, 1000)
```

You should now see new properties 
- currentLoadSteal (percentage values)
- currentLoadGuest (percentage values)
- rawCurrentLoadSteal ... ticks
- rawCurrentLoadGuest ... ticks

and also you should see it also per CPU core ... works of course JUST in linux!

If this is fine on your side, I will release a new version with the added fields. and will update the docs.

**Comment by sebhildebrandt** [comment]
@Cheyenne55 I published version 5.19.0 that includes this changes (new values can be found in function `currentLoad()`). See also updated docs here https://systeminformation.io/processes.html. 

If you have time to test it, that would be great!!!! Thank you in advance.

**Comment by sebhildebrandt** [comment]
@Cheyenne55 ... closing it for now. If you discover any issues, please feel free to reopen it.
